src/common/local_file_system.cpp

```diff
@@ -556,6 +732,12 @@ unique_ptr<FileHandle> LocalFileSystem::OpenFile(const string &path_p, uint8_t f
 	                           flags_and_attributes, NULL);
 	if (hFile == INVALID_HANDLE_VALUE) {
 		auto error = LocalFileSystem::GetLastErrorAsString();
+
+		auto better_error = AdditionalLockInfo(unicode_path);
```

Comment:
This seems to be called whenever `CreateFileW` fails (which might be for different reasons than a lock failure). Perhaps this should only be called when `GetLastError` returns `ERROR_ACCESS_DENIED`?